src/app/modules/performance/components/key-performance-areas-employee-grade/key-performance-areas-employee-grade.component.html

```diff
@@ -36,7 +36,7 @@ <h3 class="card-title prod-name avisys-table-heading">Key Performance Area Emplo
                             <div class="col-md-8">
                                 <input type="text" id="weightage" formControlName="weightage" class="form-control">
                                 <div *ngIf="keyPerformanceAreasGradeFormControl.weightage?.invalid && keyPerformanceAreasGradeFormControl.weightage?.touched">
-                                    <span class="validate-text" *ngIf="keyPerformanceAreasGradeFormControl.weightage?.errors?.required">Please enter Weightage (Only Digits)</span>
+                                    <span class="validate-text" *ngIf="keyPerformanceAreasGradeFormControl.weightage?.errors?.required">Please enter Weightage</span>
                                     <span class="validate-text" *ngIf="keyPerformanceAreasGradeFormControl.weightage?.errors?.maxlength ">Weightage exceeds max-length {{keyPerformanceAreasGradeFormControl.weightage?.errors?.maxlength.requiredLength}} (Only Digits)</span>
```

Comment:
Please check the validation message, I dont thing so there is any need of (only digits) in max-length validation.